src/modules/form/components/group/FormCard.tsx

```diff
@@ -111,7 +111,11 @@ const FormCard: React.FC<FormCardProps> = ({
         >
           {/* Card title */}
           {item.text && (
-            <Stack justifyContent='space-between' direction='row'>
+            <Stack
+              justifyContent='space-between'
+              direction={{ xs: 'column', sm: 'row' }}
+              sx={{ mb: 2 }}
```

Comment:
This looks to add quite a bit of spacing on desktop, more than necessary imo. How about: `sx={{ mb: { xs: 2, sm: 0 } }}` (I didn't know you could do that until just now! handy)

Reply:
Much better! Thank you